docs/modules/ROOT/pages/getting-started/component_deployment_lifecycle.adoc

```diff
@@ -0,0 +1,50 @@
+= Component deployment lifecycle
```

Comment:
Could we link to a CLI option in this guide once we have the getting started CLI guide? I.E. how to monitor your components via CLI?

Reply:
Great point--the way I'm planning on reorganizing the docs will definitely take care of this :+1: